### PR TITLE
Autocomplete update

### PR DESCRIPTION
**Pull request by pjsier**
Datalist was previously limited to only showing items that followed the exact same character sequence (So "1234 N Main" wouldn't pull up "1234 North Main"). Fixed by using twitter typeahead.js instead of datalist

### CODE REVIEW

**Comment by pjsier** [comment]
Good call on the point dot. I'm gonna leave "focus.point.lat" etc. the way it is because it seems like more trouble than it's worth to nest that much for two fields

File: script.js [hunk 1]
```diff
@@ -81,66 +94,68 @@ function searchAddress(submitAddr) {
...
-        searchNeighborhoods(pos, chi_json);
+      else if (url === search_url) {
+        if (data && data.features) {
+          var pos = [data.features[0].geometry.coordinates[1], data.features[0].geometry.coordinates[0]];
+          searchNeighborhoods(pos, chi_json);
+        }
       }
+    },
+    error: function(err) {
+      console.log(err);
```
Comment (karbassi) [inline script.js]: Since it's an error, consider using [`console.error()`](https://developer.mozilla.org/en-US/docs/Web/API/Console/error)

**Comment by pjsier** [comment]
What object do you pass to `console.error()`? I'm seeing you pass it an object argument, but would it be `err.message`? And [Mozilla says it's non-standard](https://developer.mozilla.org/en-US/docs/Web/API/Console/error) but is it generally used in practicality?

**Comment by karbassi** [comment]
`console.error()` is used by many developers. See here: https://github.com/search?l=javascript&q=console.error&type=Code&utf8=%E2%9C%93

It was started by Firebug, which started the whole `console.*` situation. It was later added to Firefox. When Chrome came on the scene, it had native `console.*` support.  See: https://developers.google.com/web/tools/chrome-devtools/debug/console/console-reference

I believe that `console` has become the defacto logging tool for most, if not all, javascript tools.
- [Node supports it](https://nodejs.org/api/console.html)
- JS supersets support it.

**Comment by pjsier** [comment]
Cool, that makes sense. Just pushed an update, used `console.error(err.responseText)`. Let me know if that's what you were thinking. And that works for me, it seems like "non-standard" is weirdly defined.

**Pull request merged**